completion/available/system.completion.bash

```diff
@@ -3,26 +3,30 @@
 # Loads the system's Bash completion modules.
 # If Homebrew is installed (OS X), its Bash completion modules are loaded.
 
-if [ -f /etc/bash_completion ]; then
-  . /etc/bash_completion
+if [ -r /etc/bash_completion ] ; then
+  # shellcheck disable=SC1091
+  source /etc/bash_completion
 fi
 
 # Some distribution makes use of a profile.d script to import completion.
-if [ -f /etc/profile.d/bash_completion.sh ]; then
-  . /etc/profile.d/bash_completion.sh
+if [ -r /etc/profile.d/bash_completion.sh ] ; then
+  # shellcheck disable=SC1091
+  source /etc/profile.d/bash_completion.sh
 fi
 
+if [ "$(uname -s)" == 'Darwin' ] && _command_exists brew ; then
 
-if [ $(uname) = "Darwin" ] && command -v brew &>/dev/null ; then
   BREW_PREFIX=$(brew --prefix)
 
-  if [ -f "$BREW_PREFIX"/etc/bash_completion ]; then
-    . "$BREW_PREFIX"/etc/bash_completion
+  if [ -r "$BREW_PREFIX"/etc/bash_completion ] ; then
```

Comment:
Didn't we want to get rid of this `if`? The one below should be sufficient, shouldn't it?

Reply:
[You told me to do that](https://github.com/Bash-it/bash-it/pull/1641#discussion_r467729823), I just didn't get to it yet. my bad.